example/pubspec.lock

```diff
@@ -14,7 +14,7 @@ packages:
       name: analyzer
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "1.7.2"
+    version: "1.7.1"
```

Comment:
What is the reason for downgrading the versions in this file?

Reply:
I believe this was done automatically after a `flutter pub get` I did at some point. I will trace down why this happened and get them upgraded again, but off the top of my head I only know this was not done intentionally